constants/constants.go

```diff
@@ -0,0 +1,31 @@
+package constants
```

Comment:
Two alternatives, `config` and `defaults`. Not sure if/which would be better.

Reply:
I personally dislike both of those options :sweat_smile: but I don't have a better proposal.